src/content/docs/apm/agents/ruby-agent/configuration/ruby-agent-configuration.mdx

```diff
@@ -1545,10 +1417,9 @@ If `true`, enables [cross-application tracing](/docs/agents/ruby-agent/features/
       </tbody>
     </table>
 
-  * Specify a maximum number of custom events to buffer in memory at a time.
-  * When configuring the agent for [AI monitoring](/docs/ai-monitoring/intro-to-ai-monitoring), set to max value `100000`. Ensures the agent captures the maximum amount of LLM events.
```

Comment:
This change has been added to our generator and will return the next time we run it.